doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -51,7 +51,44 @@ TODO
 
 ### Riak
 
-TODO
+Changes to Riak schema are backwards compatible with the current MongooseIM release.
+This means that in case of skipping of the migrations, only some new features (namely GDPR data retrival) won't work correctly.
```

Comment:
This means that skipping the migration will cause only some of the new features (namely GDPR data retrival) to not work correctly.